<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Location</class>
 <widget class="QDialog" name="Location">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Location</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_modifier">
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_supprimer">
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
